units/patch_intent.go

```diff
@@ -213,7 +213,7 @@ func (j *patchIntentProcessor) finishPatch(ctx context.Context, patchDoc *patch.
 		return errors.Wrap(err, "can't find patch project")
 	}
 
-	if !pref.IsEnabled() {
+	if !pref.IsEnabled() && (j.IntentType != patch.GithubIntentType || !pref.IsHidden()) {
```

Comment:
same comment about hidden for dispatching tasks

Reply:
Same answer: this is the special case